docs/pages/world-explorer.mdx

```diff
@@ -0,0 +1,85 @@
+# World Explorer
+
+World Explorer is a GUI tool designed for visually exploring and manipulating the state of worlds.
+
+## Getting started
+
+##### 1. **Install the package**
+
+```sh
+pnpm add @latticexyz/explorer
```

Comment:
Where do I install this? As part of `packages/client`? In a separate directory? 

When I tried to install and use it, I got:

```
qbzzt@Oris-MacBook-Pro client % npx @latticexyz/explorer --worldAddress 0x8d8b6b8414e1e3dcfd4168561b9be6bd3bf6ec4b  

> @latticexyz/explorer@2.2.0-main-f1d8d71e9a68f6de816a7cc1d974b9cbd78a6282 start /Users/qbzzt/run/tutorial/node_modules/.pnpm/@latticexyz+explorer@2.2.0-main-f1d8d71e9a68f6de816a7cc1d974b9cbd78a6282_@aws-sdk+client-kms@_ei4elkwfgfkqnxwvi4mpmkjbji/node_modules/@latticexyz/explorer
> node .next/standalone/packages/explorer/server.js

node:internal/modules/cjs/loader:1143
  throw err;
  ^

Error: Cannot find module '/Users/qbzzt/run/tutorial/node_modules/.pnpm/@latticexyz+explorer@2.2.0-main-f1d8d71e9a68f6de816a7cc1d974b9cbd78a6282_@aws-sdk+client-kms@_ei4elkwfgfkqnxwvi4mpmkjbji/node_modules/@latticexyz/explorer/.next/standalone/packages/explorer/server.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1140:15)
    at Module._load (node:internal/modules/cjs/loader:981:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.20.1
```
```